### PR TITLE
feat: add stream for MgvOracle

### PR DESCRIPTION
**Pull request by peterMangrove**

### CODE REVIEW

File: contracts/peterMangrove/MgvOracle/app.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+name: mangrove-oracle
```
Comment (kalabukdima) [inline contracts/peterMangrove/MgvOracle/app.yml]: Please change the dir name to also be `mangrove-oracle`. And for the username, you can just use `mangrove`

File: contracts/peterMangrove/MgvOracle/app.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+maintainer: peter@mangrove.exchange
+network: polygon-mumbai
+startBlock: 35665998
+version: 0.1.0
+contracts:
+  weth:
```
Comment (kalabukdima) [inline contracts/peterMangrove/MgvOracle/app.yml]: You should probably change it to the contact name

**Review by kalabukdima — CHANGES_REQUESTED** [review]

File: contracts/peterMangrove/MgvOracle/app.yml (hunk 2 above)
Comment (peterMangrove) [reply contracts/peterMangrove/MgvOracle/app.yml]: Good point, it has been changed

File: contracts/peterMangrove/MgvOracle/app.yml (hunk 1 above)
Comment (peterMangrove) [reply contracts/peterMangrove/MgvOracle/app.yml]: Done

**Review by kalabukdima — APPROVED** [review]

**Pull request merged**